services-camel-app/src/main/webapp/WEB-INF/solr-indexing-context.xml

```diff
@@ -92,6 +93,7 @@
                 <ref bean="setRecordDatesFilter" />
                 <ref bean="setContentTypeFilter" />
                 <ref bean="setDatastreamFilter" />
+                <ref bean="setViewBehaviorFilter" />
```

Comment:
We might not need this here, I think this pipeline is triggered when a file in a work is modified